setup.py

```diff
@@ -1,5 +1,27 @@
-from setuptools import setup, find_packages
+import sys
+import subprocess
+from setuptools import setup, find_packages, Extension
 
+LIBRARIES = ['xml2']
+SOURCES = ['bblfsh/pyuast.c']
```

Comment:
These are not constants, they shouldn't be upcase

Reply:
I've seen some uppercased module-globals on Python that were not constants but it doesn't seem to be a consensus, PEP8 says that constants are uppercase but doesn't say anything about module (mutable) globals but I think that making them lowercase avoids confusion, yes.